### ISSUE TITLE
Work Package Progress - Convert Expected Activities and Deliverables to Displayed as Checkboxes

### ISSUE THREAD

**Issue by RChandler234** [Northeastern-Electric-Racing/FinishLine#186]
### Description

On the work package view page change expected activities and deliverables from being displayed as bullet points to displayed as checkboxes. When a bullet is completed, the checkbox is filled in and the text is formatted as strike-through. See the mockups below. This is just the superficial front-end change, checking/unchecking the boxes shouldn't do anything yet, that'll be addressed in a future ticket. This ticket is dependent on #199 being done.

### Background

This ticket is contributing to the Drive Progress Based on Work Package Definition Checklist epic. This epic is meant to simplify the workflow for people using Finishline and automate the process of the progress being calculated for a work package.

### Proposed Solution

We should have a pre-existing checklist React component from the Risk-Log epic, but it's very specific to the risk log. I'd suggest making a new checklist component to replace the bullet list component being used rn. It should follow all the parameters above and can be based on the risk log checklist. Making a new component can be tricky, and we'll usually be a little more nit picky on the code since it has potential to be re-used in a lot of spots (fair warning), so feel free to ask questions early and often.

Tl;dr: should be a straightforward frontend change, don't be afraid to ask any questions in #software

### Mocks

![image](https://user-images.githubusercontent.com/29521172/189247885-5270fbd4-b00f-4e49-8f88-a46d8e7c2984.png)
